tests/integration/targets/nxos_interfaces/tasks/cli.yaml

```diff
@@ -1,21 +1,27 @@
 ---
 - name: collect common test cases
   find:
-    paths: '{{ role_path }}/tests/cli'
+    paths: '{{ role_path }}/tests/common'
     patterns: '{{ testcase }}.yaml'
   connection: local
   register: test_cases
 
+- name: collect cli test cases
```

Comment:
All the tests have been moved under the common folder. Do we still need to keep this task?

Reply:
This is consistent with the rest of the modules, and makes adding transport-specific tests easy if it becomes important later. I'm not saying it's necessary or even likely that we will have transport-specific tests, but the cost is practically nil for not having to fiddle with these files again if we do.

Reply:
Yeah, makes sense.